mars/optimization/physical/numexpr.py

```diff
@@ -100,35 +181,74 @@ def is_available(cls) -> bool:
     def optimize(self):
         fuses = []
         explored = set()
+        cached_can_fuse = functools.lru_cache(maxsize=None)(_can_fuse)
```

Comment:
Why not just add annotation `lru_cache` to `_can_fuse` directly?

Reply:
To avoid caching chunks in global and is slightly faster.

- `lru_cache` in optimize: unlimited cache size without lru.
- `lru_cache` to `_can_fuse`: fixed cache size with lru.

The chunks may be unused after the subtask execution, we don't need to cache it in global.